src/js/extensions/paste.js

```diff
@@ -60,7 +60,10 @@
             [new RegExp(/<\/?o:[a-z]*>/gi), ''],
 
             // Microsoft Word adds some special elements around list items
-            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1']
+            [new RegExp(/<!\[if !supportLists\]>(((?!<!).)*)<!\[endif]\>/gi), '$1'],
+
+            // remove font tag
+            [new RegExp(/<font[^><]*>|<.font[^><]*>/gi), '']
```

Comment:
Yes, I understand your concern. 

For now, I just passed the custom replacement just as you said.

But as I said too, the cleanPastedHTML attribute, in my mind, makes sense that every string's style could be cleaned, I did not expect that string colour was passed to the field.

I'll just drop here the font tag, copied from open office:

`
<font color="#ff3333">String
test</font>
`

There's no way to distinguish from Word or other editor, I guess.